filebeat/harvester/reader.go

```diff
@@ -53,6 +55,7 @@ func newLogFileReader(
 		config:       config,
 		lastTimeRead: time.Now(),
 		backoff:      config.backoffDuration,
+		done:         done,
```

Comment:
I wonder if channel is required. If file get's closed by another thread (in stop method), we should get some error, right?